R/epiprocess-data.R

```diff
@@ -66,87 +74,117 @@
-#' by JHU-CSSE and Downloaded from the CMU Delphi COVIDcast Epidata API.
-#' This data set covers the period from June 2020 to May 2021, and is
-#' used in the epiprocess Vignette on Outliers.
-#'
-#' @source This object contains a modified part of the \href{https://github.com/CSSEGISandData/COVID-19}{COVID-19 Data Repository by the Center for Systems Science and Engineering (CSSE) at Johns Hopkins University} as \href{https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/jhu-csse.html}{republished in the COVIDcast Epidata API}. This data set is licensed under the terms of the
-#' \href{https://creativecommons.org/licenses/by/4.0/}{Creative Commons Attribution 4.0 International license}
-#' by the Johns Hopkins University on behalf of its Center for Systems Science in Engineering.
-#' Copyright Johns Hopkins University 2020.
+#' * \href{https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/jhu-csse.html}{From the COVIDcast Epidata API}:
+#'   These signals are taken directly from the JHU CSSE \href{https://github.com/CSSEGISandData/COVID-19}{COVID-19 GitHub repository} without changes.
+#' * Furthermore, the data has been limited to a very small number of rows,
+#'   formatted into a tibble, and the signal names slightly altered.
 "covid_incidence_outliers"
 
-#' COVID Cumulative Cases Example
-#'
-#' Data set for 4 states containing COVID-19 Cumulative Cases as reported
-#' by JHU-CSSE and downloaded from the CMU Delphi COVIDcast Epidata API.
-#' This data set covers the period from March 2020 to January 2022, and is
-#' used in the epiprocess Vignette on ... .
-#'
+#' Subset of COVID-19 Cumulative Cases from 4 states
+#' @description
+#' Data set for 4 states containing COVID-19 Cumulative Cases as reported by
+#' JHU-CSSE and downloaded from the CMU Delphi COVIDcast Epidata API. This
+#' data set covers the period from March 2020 to January 2022, and is limited
+#' to California, Florida, New York, and Texas.
+#'
+#' It is used in the {epiprocess} "Getting Started" vignette.
+#'
+#' @format An object of class \code{tbl_df} (inherits from \code{tbl}, \code{data.frame}) with 2808 rows and 15 columns.
+#' @section Data dictionary:
+#' A tibble with 2,808 rows and 15 variables:
+#' \describe{
+#'   \item{geo_value}{the geographic value associated with each row of measurements.}
+#'   \item{signal}{name of metric, derived from upstream data.}
+#'   \item{source}{name of upstream data source.}
+#'   \item{geo_type}{spatial resolution of the signal.}
+#'   \item{time_type}{temporal resolution of the signal.}
+#'   \item{time_value}{the time value associated with each row of measurements.}
+#'   \item{direction}{trend classifier (+1 -> increasing, 0 -> steady or not determined, -1 -> decreasing).}
+#'   \item{issue}{time unit (e.g., date) when the signal data were published.}
+#'   \item{lag}{time delta (e.g. days) between when the underlying events happened and when the data were published.}
```

Comment:
these are a bit imprecise but I don't know if it's worth wordy precise explanations

Reply:
Yeah, I think it's fine.